### ISSUE TITLE
Floating point exception when configuring lost of LEDs per side

### ISSUE THREAD

**Issue by Chaphasilor** [hyperion-project/hyperion.ng#1084]
- [x] I confirm that this is an issue rather than a question.

## Bug report

I use a ws281x LED strip with 144 LEDs on ~1.5m. That's most likely way more than necessary, but given the short length of the strip, I still would like to have it only show a single side (e.g. bottom side).

If I configure less than 144 LEDs in the layout settings, my strip will only light up partially; the only workaround for the issue described below so far is splitting the LEDs onto all for sides, but this doesn't result in correct ambilight, naturally.

#### Steps to reproduce

In LED Layout settings, set a value >80 for any single side (e.g. bottom)

#### What is expected?

The program 'renders' the correct amount of LEDs

#### What is actually happening?

The program crashes with a floating point exception. I'm guessing this is due to the way LED positions are calculated ([see the docs about it here](https://docs.hyperion-project.org/en/user/advanced/Advanced.html#led-layout)). The more LEDs you add per side, the more decimal places you need to represent their positions. Too much decimal places => floating point exception.

Here's the log recorded with `hyperiond -d`:  
[hyperion.log](https://github.com/hyperion-project/hyperion.ng/files/5507107/hyperion.log)

#### System

```
Hyperion Server: 
- Build:       (HEAD detached at 08909b8a) (Paulchen-Panther-75963ca9/08909b8a-1600115237)
- Build time:  Sep 14 2020 21:04:51
- Git Remote:  https://github.com/hyperion-project/hyperion.ng
- Version:     2.0.0-alpha.8
- UI Lang:     auto (BrowserLang: en-US)
- UI Access:   default
- Avail Capt:  dispmanx,v4l2,framebuffer,qt

Hyperion Server OS: 
- Distribution: Open Source Media Center
- Arch:         arm
- Kernel:       linux (4.19.122-1-osmc (WS: 32))
- Browser:      Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4287.0 Safari/537.36 Edg/88.0.673.0
```

**Comment by Lord-Grey** [comment]
@Chaphasilor Thanks for reporting.
The issue is already known #909 and has been already fixed by #1047.

**Comment by Lord-Grey** [comment]
Closed as Duplicate

**Comment by Chaphasilor** [comment]
> @Chaphasilor Thanks for reporting.
> The issue is already known #909 and has been already fixed by #1047.

Great! Thanks a lot :D
